README.md

```diff
@@ -25,15 +25,23 @@ In this challenge use `Test Driven Development` to build a RESTful API using Nod
 Demonstrate your understanding of this week's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
 1. In Jest, what are the differences between `describe()` and `it()` globals, and what are good uses for them?
+
+Describe breaks your test suite into different components, where as it is where you actually perform the individual tests. 
+
 1. What is the point of `Test Driven Development`? What do you think about this approach?
+
+It is useul because it allows us to create web apps that are testable, because we are writing the tests beore the code.This makes sure they are testable apps because we are writing the tests before hand, not after. It will also save you time and money down the line. I think its a good approach.
```

Comment:
Good answers here Michael 👍 , there are a couple of typos and I might add to your answer to question #1 that `describe()` blocks group related tests together but your reasoning on question #2 is really good. 
```suggestion
It is useful because it allows us to create web apps that are testable, because we are writing the tests before the code. This makes sure they are testable apps because we are writing the tests beforehand, not after. It will also save you time and money down the line. I think its a good approach.
```